src/App.tsx

```diff
@@ -1,14 +1,77 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [isListLoaded, setIsListLoaded] = useState(false);
+  const [filterBy, setFilterBy] = useState('all');
```

Comment:
Why didn't you fix it? 
![image](https://user-images.githubusercontent.com/99403198/195811861-f098c2c7-5c04-4f21-b017-6124c36ddda6.png)